problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -17,20 +18,57 @@ function MentoringWithStepsBlock(runtime, element) {
         }
     }
 
-    var activeStep = $('.mentoring', element).data('active-step');
+    var activeStepIndex = $('.mentoring', element).data('active-step');
```

Comment:
This is kind of leftover from last PR, I left it in on the basis that this is a cleanup.